### PR TITLE
fix(review): issues from sprint review

### PR DESCRIPTION
**Pull request by PriscilliaAmmeux**
# Description
What are changes related to ? fix tthe return of review basti 

# Linked Issues
What are the issues fixed by this pull request ?#834

# Changes
What does it change ?
Is is a breaking change ?
Is is a new feature ?
Is is a patch, fix, hotfix ?

# Screenshots
Put screenshots (if any)

# Tests
How has it been tested ?

- [ x] Manually
- [ ] Automated tests
- [ ] QA
- [ ] Other

# Additional information
Precise any other information

### CODE REVIEW

File: yaki_mobile/lib/presentation/displaydata/status_page_utils.dart [hunk 1]
```diff
@@ -5,7 +5,7 @@ import 'package:yaki/domain/entities/declaration_status.dart';
 ///  Enum text (values) match the status format to be send to the API.
 enum StatusEnum {
   remote('remote'),
-  onSite('on site'),
+  office('office'),
```
Comment (LaurentMag) [inline yaki_mobile/lib/presentation/displaydata/status_page_utils.dart:8]: We need to use onSite('on site'), and not office('office'), as the value is used by the backend and we do key - value conversion (with toCamelCase function)

File: yaki_mobile/assets/translations/fr.json [hunk 2]
```diff
@@ -103,7 +102,8 @@
   "IWorkthisMorning": "Je travaille ce",
   "IWorkthisAfternoon": "Je travaille cet",
   "Iam": "Je suis",
-  "office": "Au Bureau",
+  "OnSite": "Sur site",
```
Comment (LaurentMag) [inline yaki_mobile/assets/translations/fr.json:105]: Was perfectly fine to keep "Au Bureau" just the trad key need will remain "onSite" (camelCase)

File: yaki_mobile/lib/presentation/features/declaration/declaration_half_day_end.dart [hunk 3]
```diff
@@ -69,14 +69,14 @@ class DeclarationHalfDayEnd extends ConsumerWidget {
                           height: 112,
                         ),
                         title: tr("Iam"),
-                        subtitle: tr(StatusEnum.onSite.name),
+                        subtitle: tr(StatusEnum.office.name),
```
Comment (LaurentMag) [inline yaki_mobile/lib/presentation/features/declaration/declaration_half_day_end.dart:72]: StatusEnum.onSite.name, not StatusEnum.office.name

File: yaki_mobile/lib/presentation/features/declaration/declaration_half_day_end.dart [hunk 4]
```diff
@@ -69,14 +69,14 @@ class DeclarationHalfDayEnd extends ConsumerWidget {
...
                         onSelectionChanged: (selected) {
                           onPress(
                             ref: ref,
                             declarationMode:
                                 DeclarationPaths.fromText(declarationMode),
                             teamList: teamList,
-                            buttonValue: StatusEnum.remote,
+                            buttonValue: StatusEnum.office,
```
Comment (LaurentMag) [inline yaki_mobile/lib/presentation/features/declaration/declaration_half_day_end.dart:79]: StatusEnum.onSite.name, not StatusEnum.office.name

File: yaki_mobile/lib/presentation/features/declaration/declaration_half_day_end.dart [hunk 5]
```diff
@@ -101,7 +101,7 @@ class DeclarationHalfDayEnd extends ConsumerWidget {
                             declarationMode:
                                 DeclarationPaths.fromText(declarationMode),
                             teamList: teamList,
-                            buttonValue: StatusEnum.onSite,
+                            buttonValue: StatusEnum.remote,
```
Comment (LaurentMag) [inline yaki_mobile/lib/presentation/features/declaration/declaration_half_day_end.dart:104]: StatusEnum.onSite.name, not StatusEnum.office.name

File: yaki_mobile/lib/presentation/features/declaration/declaration_page.dart [hunk 6]
```diff
@@ -209,7 +208,7 @@ List<StatusEnum> setCardSubtitle(String declarationMode) {
   if (declarationMode == DeclarationPaths.timeOfDay.text) {
     buttonsText = [StatusEnum.morning, StatusEnum.afternoon];
   } else if (declarationMode == DeclarationPaths.fullDay.text) {
-    buttonsText = [StatusEnum.remote, StatusEnum.onSite];
+    buttonsText = [StatusEnum.office, StatusEnum.remote];
```
Comment (LaurentMag) [inline yaki_mobile/lib/presentation/features/declaration/declaration_page.dart:211]: buttonsText = [StatusEnum.onSite, StatusEnum.remote];

**Review by LaurentMag — CHANGES_REQUESTED** [review]

File: yaki_mobile/lib/presentation/features/declaration/declaration_half_day_end.dart (hunk 5 above)
Comment (PriscilliaAmmeux) [reply yaki_mobile/lib/presentation/features/declaration/declaration_half_day_end.dart:104]: it's remote and not onSite. remote existing already on file status page utils.  it's concerning remote. there was an error with onSite

**Review by LaurentMag — DISMISSED** [review]

**Review by LaurentMag — APPROVED** [review]

**Pull request merged**